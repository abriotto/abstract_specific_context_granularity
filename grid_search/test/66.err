  0%|                                                                                                                                                                                                    | 0/99 [00:00<?, ?it/s]  9%|#################                                                                                                                                                                           | 9/99 [00:00<00:01, 84.24it/s] 18%|##################################                                                                                                                                                         | 18/99 [00:00<00:00, 82.75it/s] 27%|##################################################9                                                                                                                                        | 27/99 [00:00<00:00, 77.33it/s] 35%|##################################################################1                                                                                                                        | 35/99 [00:00<00:00, 67.64it/s] 42%|###############################################################################3                                                                                                           | 42/99 [00:00<00:00, 67.84it/s] 49%|############################################################################################5                                                                                              | 49/99 [00:00<00:00, 66.35it/s] 57%|#########################################################################################################7                                                                                 | 56/99 [00:00<00:00, 62.53it/s] 64%|#######################################################################################################################                                                                    | 63/99 [00:00<00:00, 64.49it/s] 71%|####################################################################################################################################2                                                      | 70/99 [00:01<00:00, 64.23it/s] 78%|#################################################################################################################################################4                                         | 77/99 [00:01<00:00, 58.96it/s] 85%|##############################################################################################################################################################6                            | 84/99 [00:01<00:00, 61.39it/s] 94%|###############################################################################################################################################################################6           | 93/99 [00:01<00:00, 67.95it/s]100%|###########################################################################################################################################################################################| 99/99 [00:01<00:00, 67.08it/s]
  0%|                                                                                                                                                                                                    | 0/25 [00:00<?, ?it/s] 20%|#####################################6                                                                                                                                                      | 5/25 [00:00<00:00, 49.54it/s] 48%|#########################################################################################7                                                                                                 | 12/25 [00:00<00:00, 52.17it/s] 76%|##############################################################################################################################################1                                            | 19/25 [00:00<00:00, 53.11it/s]100%|###########################################################################################################################################################################################| 25/25 [00:00<00:00, 55.93it/s]
