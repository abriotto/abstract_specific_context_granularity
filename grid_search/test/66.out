# ["--attributes=3", "--values=4", "--game_size=3", "--vocab_size_factor=3", "--n_epochs=100", "--batch_size=128", "--learning_rate=0.001", "--hidden_size=256", "--temp_update=0.97", "--temperature=1"]
Creating train_ds and val_ds...

Creating test_ds...
{"loss": 0.7147110104560852, "acc": 0.4993722438812256, "length": 3.539193630218506, "mode": "train", "epoch": 1}
{"loss": 0.6954668760299683, "acc": 0.5102996230125427, "length": 1.0, "mode": "test", "epoch": 1}
{"loss": 0.7000932097434998, "acc": 0.5260801911354065, "length": 2.8155481815338135, "mode": "train", "epoch": 2}
{"loss": 0.6939632892608643, "acc": 0.5102996230125427, "length": 1.0, "mode": "test", "epoch": 2}
{"loss": 0.6938527226448059, "acc": 0.5113099217414856, "length": 1.3284871578216553, "mode": "train", "epoch": 3}
{"loss": 0.6954386234283447, "acc": 0.49719101190567017, "length": 1.0, "mode": "test", "epoch": 3}
{"loss": 0.6902521848678589, "acc": 0.5174476504325867, "length": 1.042144775390625, "mode": "train", "epoch": 4}
{"loss": 0.6974695324897766, "acc": 0.5224719047546387, "length": 1.0, "mode": "test", "epoch": 4}
{"loss": 0.6941236853599548, "acc": 0.5190626382827759, "length": 1.0247236490249634, "mode": "train", "epoch": 5}
{"loss": 0.7000628709793091, "acc": 0.4971909821033478, "length": 1.0, "mode": "test", "epoch": 5}
{"loss": 0.6891273856163025, "acc": 0.5397088527679443, "length": 1.0833799839019775, "mode": "train", "epoch": 6}
{"loss": 0.6943950653076172, "acc": 0.5103433728218079, "length": 1.016853928565979, "mode": "test", "epoch": 6}
{"loss": 0.6795493364334106, "acc": 0.5790008306503296, "length": 1.5314576625823975, "mode": "train", "epoch": 7}
{"loss": 0.6614525318145752, "acc": 0.5986210107803345, "length": 2.0449438095092773, "mode": "test", "epoch": 7}
{"loss": 0.6519132852554321, "acc": 0.6228100061416626, "length": 1.9862823486328125, "mode": "train", "epoch": 8}
{"loss": 0.6440508365631104, "acc": 0.6309667229652405, "length": 1.9943820238113403, "mode": "test", "epoch": 8}
{"loss": 0.6306682825088501, "acc": 0.6559613347053528, "length": 2.016845226287842, "mode": "train", "epoch": 9}
{"loss": 0.6162882447242737, "acc": 0.685726523399353, "length": 2.5505619049072266, "mode": "test", "epoch": 9}
{"loss": 0.6228313446044922, "acc": 0.6604865193367004, "length": 2.2008519172668457, "mode": "train", "epoch": 10}
{"loss": 0.6041489839553833, "acc": 0.6805021166801453, "length": 2.196629285812378, "mode": "test", "epoch": 10}
{"loss": 0.602513313293457, "acc": 0.6524965167045593, "length": 2.0185933113098145, "mode": "train", "epoch": 11}
{"loss": 0.6098041534423828, "acc": 0.6809886693954468, "length": 2.4775280952453613, "mode": "test", "epoch": 11}
{"loss": 0.5971166491508484, "acc": 0.6728997826576233, "length": 2.2998321056365967, "mode": "train", "epoch": 12}
{"loss": 0.6042274832725525, "acc": 0.6841845512390137, "length": 2.4831459522247314, "mode": "test", "epoch": 12}
{"loss": 0.6051735877990723, "acc": 0.6701219081878662, "length": 2.1254971027374268, "mode": "train", "epoch": 13}
{"loss": 0.6028965711593628, "acc": 0.6815601587295532, "length": 2.432584285736084, "mode": "test", "epoch": 13}
{"loss": 0.5801692605018616, "acc": 0.6824564337730408, "length": 2.4261837005615234, "mode": "train", "epoch": 14}
{"loss": 0.5876772403717041, "acc": 0.7130887508392334, "length": 2.5505619049072266, "mode": "test", "epoch": 14}
{"loss": 0.5911852717399597, "acc": 0.6870784163475037, "length": 2.3678762912750244, "mode": "train", "epoch": 15}
{"loss": 0.5809381604194641, "acc": 0.6804541349411011, "length": 2.5, "mode": "test", "epoch": 15}
{"loss": 0.5610000491142273, "acc": 0.6949166655540466, "length": 2.3837366104125977, "mode": "train", "epoch": 16}
{"loss": 0.574062705039978, "acc": 0.702869176864624, "length": 2.6853933334350586, "mode": "test", "epoch": 16}
{"loss": 0.5621479749679565, "acc": 0.7211789488792419, "length": 2.689699649810791, "mode": "train", "epoch": 17}
{"loss": 0.5657508373260498, "acc": 0.7157285213470459, "length": 2.83707857131958, "mode": "test", "epoch": 17}
{"loss": 0.5459238290786743, "acc": 0.7185579538345337, "length": 2.6992368698120117, "mode": "train", "epoch": 18}
{"loss": 0.5552585124969482, "acc": 0.7224530577659607, "length": 2.7528090476989746, "mode": "test", "epoch": 18}
{"loss": 0.5440142750740051, "acc": 0.7296898365020752, "length": 2.7069520950317383, "mode": "train", "epoch": 19}
{"loss": 0.5728349089622498, "acc": 0.7151756882667542, "length": 2.9550561904907227, "mode": "test", "epoch": 19}
{"loss": 0.5435640215873718, "acc": 0.7312938570976257, "length": 2.8406646251678467, "mode": "train", "epoch": 20}
{"loss": 0.5545556545257568, "acc": 0.7310173511505127, "length": 2.7865169048309326, "mode": "test", "epoch": 20}
{"loss": 0.5150874257087708, "acc": 0.7525410652160645, "length": 2.8357746601104736, "mode": "train", "epoch": 21}
{"loss": 0.5478861331939697, "acc": 0.743694543838501, "length": 2.971909999847412, "mode": "test", "epoch": 21}
{"loss": 0.5243427753448486, "acc": 0.7585812211036682, "length": 3.0275719165802, "mode": "train", "epoch": 22}
{"loss": 0.5290386080741882, "acc": 0.7511997222900391, "length": 2.83707857131958, "mode": "test", "epoch": 22}
{"loss": 0.5097858309745789, "acc": 0.7610235214233398, "length": 2.8675477504730225, "mode": "train", "epoch": 23}
{"loss": 0.5267825126647949, "acc": 0.7720504999160767, "length": 2.8539326190948486, "mode": "test", "epoch": 23}
{"loss": 0.49783143401145935, "acc": 0.7678474187850952, "length": 2.9544310569763184, "mode": "train", "epoch": 24}
{"loss": 0.5157926678657532, "acc": 0.7762476801872253, "length": 2.938202142715454, "mode": "test", "epoch": 24}
{"loss": 0.49905329942703247, "acc": 0.7671859264373779, "length": 2.9261999130249023, "mode": "train", "epoch": 25}
{"loss": 0.5086334943771362, "acc": 0.7576170563697815, "length": 2.971909999847412, "mode": "test", "epoch": 25}
{"loss": 0.47291144728660583, "acc": 0.7806688547134399, "length": 3.0659093856811523, "mode": "train", "epoch": 26}
{"loss": 0.5049281716346741, "acc": 0.7756812572479248, "length": 3.1573033332824707, "mode": "test", "epoch": 26}
{"loss": 0.48795023560523987, "acc": 0.7876126766204834, "length": 3.178389310836792, "mode": "train", "epoch": 27}
{"loss": 0.49323540925979614, "acc": 0.7631627917289734, "length": 3.2078652381896973, "mode": "test", "epoch": 27}
{"loss": 0.4676090180873871, "acc": 0.7875239253044128, "length": 3.2186810970306396, "mode": "train", "epoch": 28}
{"loss": 0.4892732501029968, "acc": 0.771999716758728, "length": 3.3426966667175293, "mode": "test", "epoch": 28}
{"loss": 0.4648043215274811, "acc": 0.7916712164878845, "length": 3.21958065032959, "mode": "train", "epoch": 29}
{"loss": 0.5078432559967041, "acc": 0.7665461897850037, "length": 3.3595504760742188, "mode": "test", "epoch": 29}
{"loss": 0.451231449842453, "acc": 0.7986078858375549, "length": 3.2667160034179688, "mode": "train", "epoch": 30}
{"loss": 0.47923994064331055, "acc": 0.7832987904548645, "length": 3.3089888095855713, "mode": "test", "epoch": 30}
{"loss": 0.4355311095714569, "acc": 0.8033604025840759, "length": 3.158595561981201, "mode": "train", "epoch": 31}
{"loss": 0.47163471579551697, "acc": 0.7731717824935913, "length": 3.2921347618103027, "mode": "test", "epoch": 31}
{"loss": 0.4304972290992737, "acc": 0.8074007630348206, "length": 3.213852643966675, "mode": "train", "epoch": 32}
{"loss": 0.4903106391429901, "acc": 0.7818575501441956, "length": 3.4269661903381348, "mode": "test", "epoch": 32}
{"loss": 0.44756612181663513, "acc": 0.8030964732170105, "length": 3.3214778900146484, "mode": "train", "epoch": 33}
{"loss": 0.46255117654800415, "acc": 0.7884625792503357, "length": 3.3595504760742188, "mode": "test", "epoch": 33}
{"loss": 0.4337385296821594, "acc": 0.8137788772583008, "length": 3.2692599296569824, "mode": "train", "epoch": 34}
{"loss": 0.46451932191848755, "acc": 0.7905682325363159, "length": 3.3764045238494873, "mode": "test", "epoch": 34}
{"loss": 0.4431915283203125, "acc": 0.8122876882553101, "length": 3.2983930110931396, "mode": "train", "epoch": 35}
{"loss": 0.45872604846954346, "acc": 0.7928897142410278, "length": 3.3764045238494873, "mode": "test", "epoch": 35}
{"loss": 0.4173349440097809, "acc": 0.8126148581504822, "length": 3.3197438716888428, "mode": "train", "epoch": 36}
{"loss": 0.47177958488464355, "acc": 0.7896769642829895, "length": 3.3764045238494873, "mode": "test", "epoch": 36}
{"loss": 0.4218161106109619, "acc": 0.818161129951477, "length": 3.3293488025665283, "mode": "train", "epoch": 37}
{"loss": 0.46322089433670044, "acc": 0.7868189215660095, "length": 3.3595504760742188, "mode": "test", "epoch": 37}
{"loss": 0.43282079696655273, "acc": 0.8153680562973022, "length": 3.3140947818756104, "mode": "train", "epoch": 38}
{"loss": 0.4684377610683441, "acc": 0.7851474285125732, "length": 3.4269661903381348, "mode": "test", "epoch": 38}
{"loss": 0.4171108305454254, "acc": 0.8200018405914307, "length": 3.442744731903076, "mode": "train", "epoch": 39}
{"loss": 0.4694211781024933, "acc": 0.7945834398269653, "length": 3.4269661903381348, "mode": "test", "epoch": 39}
{"loss": 0.403292179107666, "acc": 0.8169949054718018, "length": 3.433823585510254, "mode": "train", "epoch": 40}
{"loss": 0.46228331327438354, "acc": 0.7913568019866943, "length": 3.4101123809814453, "mode": "test", "epoch": 40}
{"loss": 0.4226474463939667, "acc": 0.816996693611145, "length": 3.4309260845184326, "mode": "train", "epoch": 41}
{"loss": 0.443493127822876, "acc": 0.8044441342353821, "length": 3.3932583332061768, "mode": "test", "epoch": 41}
{"loss": 0.4146372973918915, "acc": 0.8266710638999939, "length": 3.4113216400146484, "mode": "train", "epoch": 42}
{"loss": 0.4712386131286621, "acc": 0.7956256866455078, "length": 3.3595504760742188, "mode": "test", "epoch": 42}
{"loss": 0.38821133971214294, "acc": 0.8240617513656616, "length": 3.379913568496704, "mode": "train", "epoch": 43}
{"loss": 0.456115186214447, "acc": 0.7941802144050598, "length": 3.4438202381134033, "mode": "test", "epoch": 43}
{"loss": 0.39709240198135376, "acc": 0.8246223330497742, "length": 3.4780163764953613, "mode": "train", "epoch": 44}
{"loss": 0.4323127865791321, "acc": 0.8048682808876038, "length": 3.4606740474700928, "mode": "test", "epoch": 44}
{"loss": 0.40608087182044983, "acc": 0.830208957195282, "length": 3.4452903270721436, "mode": "train", "epoch": 45}
{"loss": 0.46703118085861206, "acc": 0.7968645691871643, "length": 3.3932583332061768, "mode": "test", "epoch": 45}
{"loss": 0.4269562363624573, "acc": 0.8261573314666748, "length": 3.3952174186706543, "mode": "train", "epoch": 46}
{"loss": 0.45710325241088867, "acc": 0.8061447143554688, "length": 3.3932583332061768, "mode": "test", "epoch": 46}
{"loss": 0.3996981084346771, "acc": 0.8323771953582764, "length": 3.4519972801208496, "mode": "train", "epoch": 47}
{"loss": 0.4404711425304413, "acc": 0.7998678088188171, "length": 3.4101123809814453, "mode": "test", "epoch": 47}
{"loss": 0.37412551045417786, "acc": 0.8309412002563477, "length": 3.4642655849456787, "mode": "train", "epoch": 48}
{"loss": 0.45587021112442017, "acc": 0.8086543679237366, "length": 3.4101123809814453, "mode": "test", "epoch": 48}
{"loss": 0.3663009703159332, "acc": 0.8394442200660706, "length": 3.445896863937378, "mode": "train", "epoch": 49}
{"loss": 0.463483989238739, "acc": 0.8054190874099731, "length": 3.3932583332061768, "mode": "test", "epoch": 49}
{"loss": 0.36049389839172363, "acc": 0.8409701585769653, "length": 3.406585216522217, "mode": "train", "epoch": 50}
{"loss": 0.4497196674346924, "acc": 0.8018423318862915, "length": 3.3764045238494873, "mode": "test", "epoch": 50}
{"loss": 0.3479512631893158, "acc": 0.8507740497589111, "length": 3.427638292312622, "mode": "train", "epoch": 51}
{"loss": 0.42664459347724915, "acc": 0.8196307420730591, "length": 3.4269661903381348, "mode": "test", "epoch": 51}
{"loss": 0.3685579299926758, "acc": 0.8497247695922852, "length": 3.4675726890563965, "mode": "train", "epoch": 52}
{"loss": 0.43182188272476196, "acc": 0.8200023770332336, "length": 3.4606740474700928, "mode": "test", "epoch": 52}
{"loss": 0.36347538232803345, "acc": 0.8421462178230286, "length": 3.479276180267334, "mode": "train", "epoch": 53}
{"loss": 0.43767061829566956, "acc": 0.8092778921127319, "length": 3.6460673809051514, "mode": "test", "epoch": 53}
{"loss": 0.35382866859436035, "acc": 0.8444392681121826, "length": 3.6180801391601562, "mode": "train", "epoch": 54}
{"loss": 0.4504131078720093, "acc": 0.807563841342926, "length": 3.6123595237731934, "mode": "test", "epoch": 54}
{"loss": 0.3578580319881439, "acc": 0.849502682685852, "length": 3.5709216594696045, "mode": "train", "epoch": 55}
{"loss": 0.4571966230869293, "acc": 0.8162211775779724, "length": 3.629213571548462, "mode": "test", "epoch": 55}
{"loss": 0.33256956934928894, "acc": 0.8543760776519775, "length": 3.63350772857666, "mode": "train", "epoch": 56}
{"loss": 0.4339970350265503, "acc": 0.8225664496421814, "length": 3.696629285812378, "mode": "test", "epoch": 56}
{"loss": 0.348956435918808, "acc": 0.8528726696968079, "length": 3.7164347171783447, "mode": "train", "epoch": 57}
{"loss": 0.4370962679386139, "acc": 0.8129023909568787, "length": 3.797752857208252, "mode": "test", "epoch": 57}
{"loss": 0.35249945521354675, "acc": 0.8411155939102173, "length": 3.7233738899230957, "mode": "train", "epoch": 58}
{"loss": 0.4421805143356323, "acc": 0.8231508135795593, "length": 3.764045000076294, "mode": "test", "epoch": 58}
{"loss": 0.34255000948905945, "acc": 0.8604410290718079, "length": 3.7402286529541016, "mode": "train", "epoch": 59}
{"loss": 0.4403575658798218, "acc": 0.8271840214729309, "length": 3.898876428604126, "mode": "test", "epoch": 59}
{"loss": 0.35879120230674744, "acc": 0.846182107925415, "length": 3.8057754039764404, "mode": "train", "epoch": 60}
{"loss": 0.4574968218803406, "acc": 0.8183521032333374, "length": 4.0, "mode": "test", "epoch": 60}
{"loss": 0.3251388967037201, "acc": 0.8548389673233032, "length": 3.9038753509521484, "mode": "train", "epoch": 61}
{"loss": 0.4281594157218933, "acc": 0.8202247023582458, "length": 4.0, "mode": "test", "epoch": 61}
{"loss": 0.3183939456939697, "acc": 0.8653469085693359, "length": 3.9752871990203857, "mode": "train", "epoch": 62}
{"loss": 0.4347374737262726, "acc": 0.8230338096618652, "length": 4.0, "mode": "test", "epoch": 62}
{"loss": 0.32951027154922485, "acc": 0.8592690825462341, "length": 3.9989964962005615, "mode": "train", "epoch": 63}
{"loss": 0.42976316809654236, "acc": 0.8192883729934692, "length": 4.0, "mode": "test", "epoch": 63}
{"loss": 0.3124561309814453, "acc": 0.8620367646217346, "length": 3.9959802627563477, "mode": "train", "epoch": 64}
{"loss": 0.4204981327056885, "acc": 0.8183521032333374, "length": 4.0, "mode": "test", "epoch": 64}
{"loss": 0.3269430994987488, "acc": 0.8626579642295837, "length": 3.996265172958374, "mode": "train", "epoch": 65}
{"loss": 0.42835551500320435, "acc": 0.8258426785469055, "length": 4.0, "mode": "test", "epoch": 65}
{"loss": 0.31830698251724243, "acc": 0.8698894381523132, "length": 3.998501777648926, "mode": "train", "epoch": 66}
{"loss": 0.41291433572769165, "acc": 0.8267790675163269, "length": 4.0, "mode": "test", "epoch": 66}
{"loss": 0.29231682419776917, "acc": 0.8749055862426758, "length": 3.9990732669830322, "mode": "train", "epoch": 67}
{"loss": 0.4315577745437622, "acc": 0.8249063491821289, "length": 4.0, "mode": "test", "epoch": 67}
{"loss": 0.2960026264190674, "acc": 0.872096061706543, "length": 3.9992825984954834, "mode": "train", "epoch": 68}
{"loss": 0.42067408561706543, "acc": 0.8239700794219971, "length": 4.0, "mode": "test", "epoch": 68}
{"loss": 0.2966788709163666, "acc": 0.8733543753623962, "length": 3.999828577041626, "mode": "train", "epoch": 69}
{"loss": 0.4049204885959625, "acc": 0.8342697620391846, "length": 4.0, "mode": "test", "epoch": 69}
{"loss": 0.2978328764438629, "acc": 0.8683439493179321, "length": 3.9994187355041504, "mode": "train", "epoch": 70}
{"loss": 0.3906925320625305, "acc": 0.8333333730697632, "length": 4.0, "mode": "test", "epoch": 70}
{"loss": 0.30647537112236023, "acc": 0.8764836192131042, "length": 3.9999165534973145, "mode": "train", "epoch": 71}
{"loss": 0.4045213460922241, "acc": 0.8305243253707886, "length": 4.0, "mode": "test", "epoch": 71}
{"loss": 0.289385050535202, "acc": 0.8726404309272766, "length": 3.995274782180786, "mode": "train", "epoch": 72}
{"loss": 0.3939802050590515, "acc": 0.8370785713195801, "length": 4.0, "mode": "test", "epoch": 72}
{"loss": 0.2812909781932831, "acc": 0.8817996382713318, "length": 3.999955177307129, "mode": "train", "epoch": 73}
{"loss": 0.4044274091720581, "acc": 0.8333333730697632, "length": 4.0, "mode": "test", "epoch": 73}
{"loss": 0.28611037135124207, "acc": 0.8808516263961792, "length": 3.999462604522705, "mode": "train", "epoch": 74}
{"loss": 0.4302639961242676, "acc": 0.8333333134651184, "length": 4.0, "mode": "test", "epoch": 74}
{"loss": 0.30649080872535706, "acc": 0.8827366232872009, "length": 3.999903440475464, "mode": "train", "epoch": 75}
{"loss": 0.4303956627845764, "acc": 0.8333333730697632, "length": 4.0, "mode": "test", "epoch": 75}
{"loss": 0.2845678925514221, "acc": 0.8851997256278992, "length": 3.9984591007232666, "mode": "train", "epoch": 76}
{"loss": 0.43751248717308044, "acc": 0.8230336904525757, "length": 4.0, "mode": "test", "epoch": 76}
{"loss": 0.27798303961753845, "acc": 0.8883599638938904, "length": 3.99969220161438, "mode": "train", "epoch": 77}
{"loss": 0.4366999864578247, "acc": 0.8286517858505249, "length": 4.0, "mode": "test", "epoch": 77}
{"loss": 0.28487926721572876, "acc": 0.8889915347099304, "length": 3.9999468326568604, "mode": "train", "epoch": 78}
{"loss": 0.4132035970687866, "acc": 0.8333331942558289, "length": 4.0, "mode": "test", "epoch": 78}
{"loss": 0.2680862545967102, "acc": 0.8864910006523132, "length": 3.9999840259552, "mode": "train", "epoch": 79}
{"loss": 0.4328657388687134, "acc": 0.8258426785469055, "length": 4.0, "mode": "test", "epoch": 79}
{"loss": 0.32797640562057495, "acc": 0.8724192976951599, "length": 3.999948740005493, "mode": "train", "epoch": 80}
{"loss": 0.4324449896812439, "acc": 0.8267791271209717, "length": 4.0, "mode": "test", "epoch": 80}
{"loss": 0.3048304319381714, "acc": 0.8802365660667419, "length": 3.9999582767486572, "mode": "train", "epoch": 81}
{"loss": 0.405458927154541, "acc": 0.8370786309242249, "length": 4.0, "mode": "test", "epoch": 81}
{"loss": 0.3087966740131378, "acc": 0.8817992806434631, "length": 3.9999449253082275, "mode": "train", "epoch": 82}
{"loss": 0.40770113468170166, "acc": 0.8398876190185547, "length": 4.0, "mode": "test", "epoch": 82}
{"loss": 0.27963292598724365, "acc": 0.8761711120605469, "length": 3.999952793121338, "mode": "train", "epoch": 83}
{"loss": 0.39190468192100525, "acc": 0.8380148410797119, "length": 4.0, "mode": "test", "epoch": 83}
{"loss": 0.2850281894207001, "acc": 0.8864904046058655, "length": 3.9999680519104004, "mode": "train", "epoch": 84}
{"loss": 0.42754030227661133, "acc": 0.8239699602127075, "length": 4.0, "mode": "test", "epoch": 84}
{"loss": 0.2658289074897766, "acc": 0.8874273896217346, "length": 3.9999382495880127, "mode": "train", "epoch": 85}
{"loss": 0.407632976770401, "acc": 0.8352060317993164, "length": 4.0, "mode": "test", "epoch": 85}
{"loss": 0.2867403030395508, "acc": 0.876484751701355, "length": 3.999983549118042, "mode": "train", "epoch": 86}
{"loss": 0.4152391254901886, "acc": 0.8146067261695862, "length": 4.0, "mode": "test", "epoch": 86}
{"loss": 0.29595693945884705, "acc": 0.8780483603477478, "length": 3.999988079071045, "mode": "train", "epoch": 87}
{"loss": 0.3913676142692566, "acc": 0.8389511704444885, "length": 4.0, "mode": "test", "epoch": 87}
{"loss": 0.2901115417480469, "acc": 0.8895633220672607, "length": 3.9990129470825195, "mode": "train", "epoch": 88}
{"loss": 0.3915730118751526, "acc": 0.834269642829895, "length": 4.0, "mode": "test", "epoch": 88}
{"loss": 0.25885841250419617, "acc": 0.8908690214157104, "length": 3.9999945163726807, "mode": "train", "epoch": 89}
{"loss": 0.3840283155441284, "acc": 0.8426965475082397, "length": 4.0, "mode": "test", "epoch": 89}
{"loss": 0.26153507828712463, "acc": 0.8945908546447754, "length": 3.998990535736084, "mode": "train", "epoch": 90}
{"loss": 0.396775484085083, "acc": 0.8352060317993164, "length": 4.0, "mode": "test", "epoch": 90}
{"loss": 0.2522537410259247, "acc": 0.892744779586792, "length": 3.9999845027923584, "mode": "train", "epoch": 91}
{"loss": 0.38944530487060547, "acc": 0.8436329960823059, "length": 4.0, "mode": "test", "epoch": 91}
{"loss": 0.26027223467826843, "acc": 0.8974358439445496, "length": 3.9999964237213135, "mode": "train", "epoch": 92}
{"loss": 0.3803775906562805, "acc": 0.8417602777481079, "length": 4.0, "mode": "test", "epoch": 92}
{"loss": 0.25281858444213867, "acc": 0.8993076682090759, "length": 3.9998631477355957, "mode": "train", "epoch": 93}
{"loss": 0.38163453340530396, "acc": 0.8436328768730164, "length": 4.0, "mode": "test", "epoch": 93}
{"loss": 0.24220864474773407, "acc": 0.8980600833892822, "length": 3.9999806880950928, "mode": "train", "epoch": 94}
{"loss": 0.4023599326610565, "acc": 0.8398875594139099, "length": 4.0, "mode": "test", "epoch": 94}
{"loss": 0.2609882056713104, "acc": 0.9024381041526794, "length": 3.9999756813049316, "mode": "train", "epoch": 95}
{"loss": 0.4142065942287445, "acc": 0.8370786309242249, "length": 4.0, "mode": "test", "epoch": 95}
{"loss": 0.30018940567970276, "acc": 0.8949340581893921, "length": 3.999990463256836, "mode": "train", "epoch": 96}
{"loss": 0.39743542671203613, "acc": 0.8398876190185547, "length": 4.0, "mode": "test", "epoch": 96}
{"loss": 0.26634737849235535, "acc": 0.8968098759651184, "length": 3.999981164932251, "mode": "train", "epoch": 97}
{"loss": 0.4257723093032837, "acc": 0.8277153968811035, "length": 4.0, "mode": "test", "epoch": 97}
{"loss": 0.2803981602191925, "acc": 0.8826810121536255, "length": 3.997793197631836, "mode": "train", "epoch": 98}
{"loss": 0.4080888032913208, "acc": 0.8295880556106567, "length": 4.0, "mode": "test", "epoch": 98}
{"loss": 0.2780457139015198, "acc": 0.8930249214172363, "length": 3.999393939971924, "mode": "train", "epoch": 99}
{"loss": 0.40560758113861084, "acc": 0.8286516666412354, "length": 4.0, "mode": "test", "epoch": 99}
{"loss": 0.2813093364238739, "acc": 0.8919664025306702, "length": 3.9980971813201904, "mode": "train", "epoch": 100}
{"loss": 0.4050477147102356, "acc": 0.8333333134651184, "length": 4.0, "mode": "test", "epoch": 100}
test accuracy: 0.7777776718139648
